docs/admin.md

```diff
@@ -0,0 +1,88 @@
+# Administrator Guide
+
+## Backstage Operator configuration
+
+### Context
+
+As it is described in Design doc (TODO), Backstage CR's desired state is defined using layered configuration approach, which means:
+- By default each newly created Backstage CR uses Operator scope Default Configuration
+- Which can be fully or partially overriden for particular CR instance using ConfigMap with witn name pointed in BackstageCR.spec.RawConfig 
+- Which in turn can be customized by other BackstageCR.spec fields (see Backstage API doc)
+
+Cluster Administrator may want to customize Default Configuration due to internal preferences/limitations, for example:
+- Preferences/restrictions for Backstage and|or PostgreSQL images due to  Airgapped environment.
+- Existed Taints and tolerations policy, so Backstage Pods have to be configured with certain tolerations restrictions.
+- ...
+
+Default Configuration is implemented as a ConfigMap called *backstage-default-config*, deployed on *backstage-system* namespace and mounted to Backstage controller container as a */default-config* directory.
+This config map contains the set of keys/values which maps to file names/contents in the */default-config*.
+These files contain yaml manifests of objects used by Backstage controller as an initial desired state of Backstage CR according to Backstage Operator data model.
+(TODO: link to the diagram here) 
```

Comment:
part of this PR or a followup one?

Reply:
@nickboldt 
I do not know, depends on this PR dynamic